src/components/PathwayExercise/index.js

```diff
@@ -134,7 +138,8 @@ function PathwayExercise() {
   const [showArrow, setShowArrow] = useState({ left: false, right: true });
   const currentCourse = params.courseId;
   const scrollRef = React.useRef();
-
+  const [language, setLanguage] = useState("en");
```

Comment:
Use a constant for English and refer to  this instead of `"en"` to allow for changes of codes (e.g., to `en-gb`)